stories/alert/index.stories.tsx

```diff
@@ -87,14 +87,18 @@ export const WithImage = () => (
  */
 export const WithAlertContainer = () => (
   <Alert.Container>
-    <Alert content="Alert.Container fixes my position" />
+    <Alert duration="sticky" content="Alert.Container fixes my position" />
   </Alert.Container>
 );
 
 export default {
   title: 'Components/Alert',
   component: Alert,
   parameters: {
+    /**
+     * TODO: Figure out how to get Alerts to show up on Chromatic
```

Comment:
I was unable to get Alerts to show up, despite my best efforts (https://www.chromatic.com/snapshot?appId=5f6e46f1e7081000221523cf&id=5f909030409e4d002292f8a0) and being able to solve ostensibly trickier components like modals. I think it's fine for now.